src/components/Atoms/Link/Link.js

```diff
@@ -64,6 +66,8 @@ Link.propTypes = {
   /** Link url */
   href: PropTypes.string.isRequired,
   children: PropTypes.node.isRequired,
+  /** Icon first */
```

Comment:
lol, probably don't need this comment?

Reply:
nah, just as other props had.. this comment will show as description in styleguide frontend .. but field name is explanatory anyway

Reply:
Gotcha